packages/web-app-files/tests/unit/components/SideBar/Shared/RoleItem.spec.js

```diff
@@ -0,0 +1,48 @@
+import stubs from '@/tests/unit/stubs/index.js'
+import { createLocalVue, mount } from '@vue/test-utils'
+import RoleItem from '@files/src/components/SideBar/Shared/RoleItem'
+
+const localVue = createLocalVue()
+
+const selectors = {
+  roleItem: '.roles-select-role-item',
+  roleLabel: '.roles-select-role-item .oc-text-bold',
+  roleDescription: '.roles-select-role-item .oc-m-rm'
+}
+
+const filesPersonalRoute = { name: 'files-personal' }
+
+function getWrapper(route, role) {
+  return mount(RoleItem, {
```

Comment:
if no interaction with the components is involved, please try using shallow mount.